### DISCUSSION TITLE
Does Ginan support reduced dynamic precise orbit determination of LEO ？

### DISCUSSION THREAD

**Discussion by realrxguo** [GeoscienceAustralia/ginan#95] [Q&A] [answered]
Dear Ginan team，
Does Ginan support reduced dynamic precise orbit determination of LEO ？
Thanks

**Comment by PingaNNU** [comment]
I also want to ask this question

**Comment by seballgeyer** [comment, marked as answer]
Answered here
https://github.com/GeoscienceAustralia/ginan/discussions/144
